### PR TITLE
Adding nginx scenarios

### PR DESCRIPTION
**Pull request by sebastienros**

### CODE REVIEW

**Comment by sebastienros** [comment]
Co-authored-by: @alnikola

File: docker/nginx/nginx.dockerfile [hunk 1]
```diff
@@ -1,11 +1,44 @@
...
+# ARG SERVER_SCHEME 
+ADD nginx-http.conf /etc/nginx/nginx.conf
+
+# Listening to https connections
+FROM base AS scheme-https-http
+# ARG SERVER_SCHEME
+ADD nginx-https.conf /etc/nginx/nginx.conf
+
+# Listening to h2 connections
+FROM base AS scheme-https-http2
```
Comment (alnikola) [inline docker/nginx/nginx.dockerfile:33]: It should be HTTP/2 - plaintext HTTP 1.1.

Comment (sebastienros) [reply docker/nginx/nginx.dockerfile:33]: This is ${SERVER_SCHEME}-${SERVER_PROTOCOL}
Nothing to do with upstream

Comment (sebastienros) [reply docker/nginx/nginx.dockerfile:33]: http-http2 would mean h2c for instance

Comment (alnikola) [reply docker/nginx/nginx.dockerfile:33]: I got it now. Sorry for the misunderstanding.

**Pull request merged**